### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by sallyom**
@abhgupta @damemi 
Updates to the README and template, brought the template here in line with what is in our ansible role.
To deploy, you can use `oc create/oc process/oc apply` outlined in the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,39 +6,22 @@ The `PERIOD` is a rolling timeframe during which pod, replication controller, an
...
+Once a replication controller has exceeded the `QUOTA_HOURS` limit, that RC is scaled to 0 replicas. Any RCs active for a deployment config contribute cumulatively toward that DC's quota-hour usage, and once it is exceeded all RCs connected to that DC will be scaled to 0 replicas. As a catch-all to deter users from skirting these rules, pods' quota-hour usage within a project accumulate during the rolling `PERIOD` until `QUOTA_HOURS` is met. In that case, a `force-sleep` quota is placed on the project with a hard limit on `pods=0`. This quota persists for `PROJECT_SLEEP_LENGTH`.  Upon removal of the `force-sleep` quota, services are placed in an idled state.  Project deployments will be scaled up to the pre-sleep value when the service within that project receives network traffic, using the same logic as oc idle and the origin unidling controller.
```
Comment (damemi) [inline README.md]: Everything here about RC and DC quota-hour usage actually isn't relevant anymore. We only track pod quota-hours now

**Comment by sallyom** [comment]
@damemi re-review please, thanks!

File: README.md [hunk 2]
```diff
@@ -2,43 +2,26 @@
...
+Once a project has exceeded the `QUOTA_HOURS` limit, that project's scalable resources are scaled to 0 replicas.  As a catch-all to deter users from skirting these rules, pods' quota-hour usage within a project accumulate during the rolling `PERIOD` until `QUOTA_HOURS` is met. In that case, a `force-sleep` quota is placed on the project with a hard limit on `pods=0`. This quota persists for `PROJECT_SLEEP_LENGTH`.  Upon removal of the `force-sleep` quota, services are placed in an idled state.  Project deployments will be scaled up to the pre-sleep value when the service within that project receives network traffic, using the same logic as oc idle and the origin unidling controller.
```
Comment (damemi) [inline README.md]: Probably should remove `As a catch-all to deter users from skirting these rules,`, then it's good

Comment (sallyom) [reply README.md]: done, re-re-review please!

**Pull request merged**